docs/en_US/TrainingService/PaiMode.md

```diff
@@ -92,8 +92,11 @@ Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMod
     * Required key. Set the mount path in your container used in PAI.
 * paiStoragePlugin
     * Optional key. Set the storage plugin name used in PAI. If it is not set in trial configuration, it should be set in the config file specified in `paiConfigPath` field.
+* command  
+    * Optional key. Set the commands used in PAI container.
 * paiConfigPath
     * Optional key. Set the file path of pai job configuration, the file is in yaml format.
+    If users set paiConfigPath in NNI's configuration file, the `command`, `paiStoragePlugin`, `virtualCluster`, `image`, `memoryMB`, `cpuNum`, `gpuNum` in `trial` filed will lose efficacy, and configurations from `paiConfigPath` will take effect. Notice that if users set `paiConfigPath`, NNI will check the validation of `commands` under  `taskRoles/taskRole/commands`.
```

Comment:
suggest to explain the merge rule when user specify `paiConfigPath`, for example, `jobName` is changed by NNI to become a random string?

Reply:
Some notes (limitations) I'm thinking of:

* Taskrole with name `taskrole` needs to exist.
* Only commands in `taskrole` will be wrapped with trial keeper. Other taskroles will be kept as is.
* Multi-line commands are not supported. Only the first command will be kept and wrapped with trial keeper.
* Special characters (like `"` and `\`) are not supported in commands.
* `jobName` specified in the original PAI configuration will be ignored.
* PAI configuration needs to be kept unchanged during the whole experiment. If it's changed or deleted, either accidentally or not, nni will use the updated configuration file to submit new jobs.

Although some of these "limitations" are by design, I think, it's still "good to know" for users.

Also, what is "efficacy"?

Reply:
Updated.  
1.  add explication for jobName in doc.
2.  remove hardcode `taskrole`,  traversal all taskRoles instead.
3. wrap all taskRoles' command.
4. multi-line command will be connected by `&&`.
5. use `\` to transfer meaning for `"` and `\` in command.
6. In current design, jobName in original PAI configuration will be replaced to NNI's job name,  If we add original jobName in NNI experiment, it may cause name-too-long exception in PAI.
7. only read original PAI configuration once in experiment, it will keep consistent in all trial jobs.